### ISSUE TITLE
Le mans ultimate crashes when using DXVK

### ISSUE THREAD

**Issue by Raboebie** [doitsujin/dxvk#3881]
### Software information
Le Mans Ultimate

### System information
- GPU: AMD 6900 XT
- Driver: AMD AMD Radeon RX 6900 XT (radeonsi, navi21, LLVM 16.0.6, DRM 3.57, 6.7.4-2-MANJARO)
  Driver Version: 4.6 (Compatibility Profile) Mesa 23.3.5-manjaro1.1
  OpenGL Version: 4.6
- Wine version: Proton Experimental
- DXVK version: Latest master

### Apitrace file(s)
https://drive.google.com/drive/folders/1dY9Iyx4pvQCu9giLFAGPKsQBooK4-YuI?usp=sharing

### Log files
Log file is in the same link.

The game starts with a splash screen, goes black for about 5 seconds and crashes. Disabling DXVK via PROTON_USE_WINED3D=1 allows the game to launch to the menu without issue. It does crash when loading a track though.

As per the wiki the trace files were quite small. Could be because it crashes so quickly but it did say to mention this.

**Comment by Blisto91** [comment]
Linking the previous dev comment for visibility
https://github.com/ValveSoftware/Proton/issues/7512#issuecomment-1974885668

**Comment by doitsujin** [comment]
Yeah, as mentioned in that comment this isn't really fixable in a sane way since the game just relies on being lucky.

**Comment by Raboebie** [comment]
Yeah I see, should I keep this issue open for visibility then or do you suggest we close it until the dev finally fixes the game.

**Comment by doitsujin** [comment]
might as well keep it open, just don't expect anything to happen here for a while.
